src/CloseButtonTrait.php

```diff
@@ -9,18 +9,24 @@
 use Yiisoft\Html\Html;
 use Yiisoft\Html\Tag\Base\Tag;
 
-abstract class AbstractCloseButtonWidget extends AbstractToggleWidget
+trait CloseButtonTrait
 {
-    protected ?array $closeButtonOptions = [];
-    protected string|Stringable $closeButtonLabel = '';
-    protected bool $encodeCloseButton = true;
+    private array $closeButtonOptions = [];
+    private string|Stringable $closeButtonLabel = '';
+    private bool $encodeCloseButton = true;
+    private bool $insideCloseButton = true;
```

Comment:
```suggestion
    private bool $showCloseButton = true;
```

Seems, this name is more understandable.